core/templates/pages/topic-editor-page/topic-editor-page.mainpage.html

```diff
@@ -1,37 +0,0 @@
-<!DOCTYPE html>
-<html ng-app="oppia" itemscope itemtype="http://schema.org/Organization">
-  <head>
-    @load('base-components/header.template.html', {"title": "Oppia"})
-  </head>
-  <body>
-    <oppia-root>
-      <div ng-controller="Base">
-        <oppia-base-content>
-          <navbar-breadcrumb>
-            <oppia-topic-editor-navbar-breadcrumb>
-            </oppia-topic-editor-navbar-breadcrumb>
-          </navbar-breadcrumb>
-
-          <nav-options>
-            <oppia-topic-editor-navbar>
-            </oppia-topic-editor-navbar>
-          </nav-options>
-
-          <mobile-nav-options>
-            <oppia-topic-editor-navbar>
-            </oppia-topic-editor-navbar>
-          </mobile-nav-options>
-
-          <content>
-            <oppia-topic-editor-page></oppia-topic-editor-page>
-          </content>
-
-          <page-footer></page-footer>
-        </oppia-base-content>
-      </div>
-
-    </oppia-root>
-    @load('pages/footer_js_libs.html')
-    <script src="/third_party/static/MathJax-2.7.5/MathJax.js?config=default"></script>
```

Comment:
@hrshkshri  Do we need to add this in the root component?

Reply:
Yeah, I believe we must add this.
Thanks.

Reply:
Done